### PR TITLE
Fix an issue where push would only load json data

### PR DESCRIPTION
**Pull request by Blokje5**
Something I noticed while looking at the push command: It would only load JSON files as data, not YAML, which is the current expected behaviour.

### CODE REVIEW

File: policy/store.go [hunk 1]
```diff
@@ -26,6 +26,33 @@ func StoreFromDataFiles(paths []string) (storage.Store, error) {
...
+// at the given path. Any JSON or YAML document
+// could be a valid document.
+func ReadDataFiles(path string) ([]string, error) {
+	var data []string
+	err := filepath.Walk(path, func(currentPath string, info os.FileInfo, err error) error {
+		if err != nil {
+			return fmt.Errorf("walk path: %w", err)
+		}
+
+		if info.IsDir() {
```
Comment (jpreese) [inline policy/store.go:39]: nit: the `.git` dir is pretty common when using conftest, there's probably value in using `filepath.SkipDir` if the path is .git'. We'll still want the `return nil`, as SkipDir will ignore the entire directory, which is desired for certain directories.

```go
if fileInfo.IsDir() && fileInfo.Name() == ".git" {
  return filepath.SkipDir
}
```

File: policy/store.go [hunk 2]
```diff
@@ -26,6 +26,33 @@ func StoreFromDataFiles(paths []string) (storage.Store, error) {
...
+			return nil
+		}
+
+		if filepath.Ext(currentPath) == ".json" || filepath.Ext(currentPath) == ".yaml" {
```
Comment (jpreese) [inline policy/store.go:43]: nit: `.yml` would also be valuable to catch

Comment (Blokje5) [reply policy/store.go]: Nice catch! I always forget `.yml` is a thing

**Comment by Blokje5** [comment]
Thanks for the review, I have updated the PR based on your comments.

**Pull request merged**